index.html

```diff
@@ -0,0 +1,87 @@
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> -->
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12">
+          <h1 id="title" class="display-3">Title</h1>
+
+        </div>
+        <p id="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere laudantium quaerat ipsa quidem dolores fugit enim saepe blanditiis id deleniti odit tempora corporis consequuntur libero nobis incidunt, ut neque nisi!</p>
+        <div class="col-md-12 mb-3">
+          <form action="" id="survey-form" class="">
+            <div class="col-md-12 mb-3">
+              <label for="name" id="name-label">
+                <input type="text" placeholder="Your Name..." id="name" required class="form-control">
+              </label>
+            </div>
+            <div class="col-md-12 mb-3">
+              <label for="email" id="email-lable">
+                <input type="email" placeholder="Your email..." id="email" required class="form-control">
+        
+              </label>
+            </div>
+            <div class="col-md-12 mb-3">
+
+            </div>
+            <div class="col-md-12 mb-3">
+              <label for="number">
+                <input type="number" id="number" placeholder="Your Number..." min="0" max="100" class="form-control">
+              </label>
+            </div>
+            <div class="col-md-12 mb-3">
+              <select class="form-select">
+                <option value="First Class"> First Class</option>
+                <option value="Seconde Class"> Second Class</option>
+                <option value="Second Class"> Second Class</option>
+             
+              </select>
+            </div>
+            <div class="col-md-12 mb-3">
+              <input class="form-check" type="checkbox" name="gender" value="Male">
+              <input class="form-check" type="checkbox" name="gender" value="Female">
+            </div>
+            <div class="col-md-12 mb-3">
+              <textarea class="form-control" name="desc" id="desc" cols="30" rows="10"></textarea>
+            </div>
+            
+            
+              
+             
+      
+              
+                <button class="btn btn-primary" id="submit">Submit</button>
+            
+          </form>
+        </div>
+          
+        </div>
+      
+      </div>
+      
+    
+    
+    <!--
+     
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
It's better to clean your code of unnecessary comments.

Reply:
Nice work being suggestive😉 But we can still improve this comment by adding reasons why we want the student to "remove unnecessary comments from their code" please see the first comment and go through this lesson on [how to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)🤗